convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,257 @@
+import tempfile
+from pathlib import Path
+from uuid import uuid4
+
+import requests
+from botocore.exceptions import ClientError
+from fastapi import HTTPException
+from tenant_dependency import TenantData
+
+from src.config import DEFAULT_PAGE_BORDER_OFFSET, settings
+from src.label_studio_to_badgerdoc.badgerdoc_format.annotation_converter import (
+    AnnotationConverter,
+)
+from src.label_studio_to_badgerdoc.badgerdoc_format.badgerdoc_format import (
+    BadgerdocFormat,
+)
+from src.label_studio_to_badgerdoc.badgerdoc_format.pdf_renderer import (
+    PDFRenderer,
+)
+from src.label_studio_to_badgerdoc.badgerdoc_format.plain_text_converter import (
+    TextToBadgerdocTokensConverter,
+)
+from src.label_studio_to_badgerdoc.models.label_studio_models import (
+    LabelStudioModel,
+    S3Path,
+)
+from src.logger import get_logger
+
+LOGGER = get_logger(__file__)
+
+
+class LabelStudioToBDConvertUseCase:
+    def __init__(
+        self,
+        s3_client,
+        current_tenant: str,
+        token_data: TenantData,
+        s3_input_annotation: S3Path,
+        s3_output_bucket: str,
+        s3_output_file_id: int,
```

Comment:
I am not confident keeping connection to S3 in this class. If you need s3 related args, just take them in corresponding method

Reply:
We make at least 3 requests to other microservices in this class methods.
I think it is more convenient to keep attributes needed for this request inside class instance attributes, rather then pass it every time

Reply:
I mean that we mix up 2 separate logics here and bound the class with this args rather than pass them to methods

Reply:
I would to prefer to keep it this way.
Right the logic is that we create instance of this class with all data it needs and run "execute" function.
Then "execute" function orchestrates all logic